routes/web.php

```diff
@@ -353,12 +353,17 @@ function (): void {
                             ->middleware('can:view,application')
                             ->name('manager.applications.show');
 
-                        /* View Applicant Profile */
-                        Route::get('jobs/{jobPoster}/applicants/{applicant}', 'ApplicantProfileController@show')
+                        /* View Applicant Profile with Job */
+                        Route::get('jobs/applicants/{applicant}', 'ApplicantProfileController@showWithJob')
```

Comment:
I don't think you meant to take out {jobPoster} here.

Reply:
Yup :p